pages/pipelines/secrets.md.erb

```diff
@@ -11,57 +11,56 @@ A best practice for secret storage is to use your own secrets storage service, s
 There are also various [Buildkite Plugins](/docs/plugins) that integrate reading
 and exposing secrets to your build steps using secrets storage services.
 
-## Storing secrets in environment hooks
+## Exporting secrets with environment hooks
```

Comment:
Exporting is a definitely better than Storing here, but I wonder if there is a third option? 

> Injecting secrets from an environment hook?

> Securely accessing secrets from ...

Reply:
I am not strongly in favor of any one of them. I just really disliked "storing" for this particular case. If you've got a preference though, I'm happy to follow that.

Reply:
Nothing strong. I think the direction of hook -> step implied with Exporting is better than variable -> hook, ie it's closer to what folks are trying to do. :man_shrugging: